### PR TITLE
net: tcp: net_tcp_parse_opts: Convert MSS value to host byte order

### PR DESCRIPTION
**Pull request by pfalcon**
Signed-off-by: Paul Sokolovsky <paul.sokolovsky@linaro.org>

### CODE REVIEW

**Comment by pfalcon** [comment]
D'oh.

File: subsys/net/ip/tcp.c [hunk 1]
```diff
@@ -1289,13 +1289,16 @@ int net_tcp_parse_opts(struct net_pkt *pkt, int opt_totlen,
 		}
 
 		switch (opt) {
-		case NET_TCP_MSS_OPT:
+		case NET_TCP_MSS_OPT: {
+			u16_t mss;
 			if (optlen != 2) {
 				goto error;
 			}
 			frag = net_frag_read(frag, pos, &pos,
```
Comment (jukkar) [inline subsys/net/ip/tcp.c]: You could use net_frag_read_be16() here directly.

Comment (pfalcon) [reply subsys/net/ip/tcp.c]: Indeed, fixed.

**Comment by pfalcon** [comment]
@jukkar : Finally passed CI.

**Review by jukkar — APPROVED** [review]

**Pull request merged**